src/applications/vaos/appointment-list/components/ConfirmedAppointmentDetailsPage/tests/StatusAlert.unit.spec.js

```diff
@@ -157,6 +159,29 @@ describe('VAOS <StatusAlert> component with After visit summary link', () => {
     expect(screen.baseElement).to.not.contain('.usa-alert-warning');
     expect(screen.queryByTestId('after-vist-summary-link')).to.exist;
   });
+  it('Should display after visit summary link error message', async () => {
+    const appointment = {
+      ...appointmentData,
+      kind: 'clinic',
+      status: 'booked',
+      avsPath: 'Error retrieving AVS link',
+      vaos: {
+        isUpcomingAppointment: false,
+        isPastAppointment: true,
+      },
+    };
```

Comment:
There is a MockAppointment class in unit-test-helpers.js. The intent of this class is to define what an appointment looks like 1 time instead of each unit test having to define it. Getters and Setters can be added as needed to take care of common logic duplicated in out unit tests. Ex. date formatting etc. Thoughts???

Reply:
@vbahinwillit - I didn't realize a class was created, thank you. I will update the tests